cdk/cdk-resources/cloud-backup-schedule/src/index.ts

```diff
@@ -109,6 +123,8 @@ export function toJson_CfnCloudBackupScheduleProps(obj: CfnCloudBackupSchedulePr
   if (obj === undefined) { return undefined; }
   const result = {
     'Id': obj.id,
+    'ProjectId': obj.projectId,
+    'ClusterName': obj.clusterName,
```

Comment:
Did we release the CFN resource with the fix?

Reply:
hi andrea! i have a list of things to fix, once all this are published, im going to share the fix we have to do on the CFN